### PR TITLE
Better link CLI and environment variables docs

### PR DESCRIPTION
**Pull request by flaurida**

### CODE REVIEW

**Comment by flaurida** [comment]
done

File: docs/configuration.md [hunk 1]
```diff
@@ -56,7 +76,7 @@ Another consideration is if you are testing a website where elements appear befo
 
 `QAW_URL` sets the URL for all tests to run against. By default each test will run against the URL where it was recorded, unless a [different URL is set for that specific test](#qaw_testname_url).
 
-For example, if you set `QAW_URL` in [your CI](set_up_ci) workflow file, all tests will run against that URL by default:
+For example, if you set `QAW_URL` in [your CI](set_up_ci) workflow file, all tests will run against that URL by default. Here's an example (yours may look slightly different depending on your CI provider):
```
Comment (jperl) [inline docs/configuration.md]: I don't think the qualifier text is necessary `(yours may look slightly different depending on your CI provider)`

Comment (flaurida) [reply docs/configuration.md]: I'm trying to be as explicit as possible

Comment (jperl) [reply docs/configuration.md]: Ok

**Pull request merged**